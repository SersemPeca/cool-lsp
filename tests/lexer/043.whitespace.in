(* various forms of whitespace *)
0
        (* blanks *)
1
      (* carriage return *)
3
	(* tab *)
4
5
