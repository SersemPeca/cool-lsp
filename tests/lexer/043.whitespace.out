#2 INT_CONST 0
#4 INT_CONST 1
#6 INT_CONST 3
#8 INT_CONST 4
#9 INT_CONST 5
